apps/frontend/src/routes/chat/+page.svelte

```diff
@@ -20,429 +16,46 @@
-						</button>
-						
-						<button 
-							class="p-6 text-left border border-gray-200 dark:border-gray-700 rounded-xl hover:border-gray-300 dark:hover:border-gray-600 hover:shadow-md transition-all duration-200 bg-white dark:bg-gray-800"
-							onclick={() => current_input = "Help me write and improve some text"}
-						>
-							<h3 class="font-medium text-gray-900 dark:text-white mb-2">Writing Assistance</h3>
-							<p class="text-sm text-gray-600 dark:text-gray-400">Draft, edit, or improve emails, documents, and more</p>
-						</button>
-						
-						<button 
-							class="p-6 text-left border border-gray-200 dark:border-gray-700 rounded-xl hover:border-gray-300 dark:hover:border-gray-600 hover:shadow-md transition-all duration-200 bg-white dark:bg-gray-800"
-							onclick={() => current_input = "Explain a complex topic in simple terms"}
-						>
-							<h3 class="font-medium text-gray-900 dark:text-white mb-2">Learn Something New</h3>
-							<p class="text-sm text-gray-600 dark:text-gray-400">Get clear explanations on topics you're curious about</p>
-						</button>
-						
-						<button 
-							class="p-6 text-left border border-gray-200 dark:border-gray-700 rounded-xl hover:border-gray-300 dark:hover:border-gray-600 hover:shadow-md transition-all duration-200 bg-white dark:bg-gray-800"
-							onclick={() => current_input = "Help me analyze and solve a problem"}
-						>
-							<h3 class="font-medium text-gray-900 dark:text-white mb-2">Problem Solving</h3>
-							<p class="text-sm text-gray-600 dark:text-gray-400">Break down challenges and find solutions together</p>
-						</button>
-					</div>
-				</div>
-			</div>
-			
-		</div>
-	</div>
-	
-	<div class="fixed bottom-0 left-0 right-0 bg-white dark:bg-gray-900 border-t border-gray-200 dark:border-gray-700">
-		<div class="w-full max-w-4xl mx-auto px-4 py-4">
-			<form id="input_form" onsubmit={inputSubmit}>
-				<Textarea
-					id="user-input"
-					disabled={is_streaming}
-					placeholder="Message..."
-					rows={2}
-					name="message"
-					bind:value={current_input}
-					clearable
-					class="focus:ring-1 focus:ring-gray-300 focus:border-gray-300 transition-all duration-200 border-gray-300 dark:border-gray-600"
-					onkeypress={(event) => {
-						if (event.key === 'Enter' && event.shiftKey === false) inputSubmit();
-					}}
-				>
-					{#snippet footer()}
-						<div class="flex items-center justify-end">
-							<Button 
-								type="submit" 
-								disabled={is_streaming}
-								class="bg-gray-900 hover:bg-gray-800 dark:bg-gray-100 dark:hover:bg-gray-200 dark:text-gray-900 text-white border-0 shadow-sm transition-all duration-200"
-							>
-								Send
-								{#if is_streaming}
-									<Spinner class="ms-2" size="4" color="white" />
-								{/if}
-							</Button>
-						</div>
-					{/snippet}
-				</Textarea>
-			</form>
-		</div>
-	</div>
+{#if isLoading}
+	<ChatSkeleton />
```

Comment:
Not quite sure whether `ChatSkeleton` is the right name for this. Perhaps a functional title, like `ChatLoader` is more appropriate?

Eventually, I want to factor this out as well -- so users calling our API don't have to worry about it.